pkcs11/_pkcs11.pyx

```diff
@@ -1550,6 +1550,19 @@ cdef class lib:
         except StopIteration:
             return token
 
+    def wait_for_slot_event(self, blocking=False):
+        cdef CK_FLAGS flag = 0
+        cdef CK_SLOT_ID slot_id
+
+        if not blocking:
+            flag = 1
+
+        rv = _funclist.C_WaitForSlotEvent(flag, & slot_id, NULL)
```

Comment:
Remove extra space between & and s

Reply:
You also want to drop the GIL while waiting here, else we learned you won't be able to use this method with asyncio. Have a look at the `with nogil` context blocks.